### PR TITLE
added stuff

### PR DESCRIPTION
**Pull request by vsluo**
hey
u know..

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,29 @@ Reading instructions:
...
+concatenated = rbind(mydata1, mydata2)
+
+# or with dplyr
+library(dplyr)
+concatenated = row_bind(mydata1, mydata2)
+```
 
-Save data to disk
+#### Save data to disk ####
 -----------------
```
Comment (asnr) [inline README.md]: Remove the underlining hyphens "`-`", which indicate a 2nd level header (`<h2>` in HTML) and conflicts with the 4th level header `####` symbol.

File: README.md [hunk 2]
```diff
@@ -155,29 +193,20 @@ dups = duplicated(mydata$myvar)
...
------------
-
-#### concatenate datasets ####
+`proc print`
+---------------
 
 ```SAS
-data concatenated;
-    set mydata1 mydata2;
+`proc print data=mydata (obs=6);
```
Comment (asnr) [inline README.md]: Remove the leading back tick: `

File: README.md [hunk 3]
```diff
@@ -199,6 +228,35 @@ run;
...
+
+`proc format`
+---------------
+#### transform values according to a format ####
+```SAS
+proc format;
+value agefmt
+15-<21 = "15-20"
+21-<25 = "21-24"
+25-high "25+"
```
Comment (asnr) [inline README.md]: Missing an `=`?

File: README.md [hunk 4]
```diff
@@ -199,6 +228,35 @@ run;
...
+run;
+data out_ds;
+set in_ds;
+age_grouped=put(age,agefmt.);
+
+run;
+```
+
+```r
+# with plyr
```
Comment (asnr) [inline README.md]: I think using `plyr` here is overkill. Simply

``` r
out_ds$age_f = cut(out_ds$age, ...)
```

is more idiomatic.

**Comment by asnr** [comment]
Thanks for the additions, I've added some specific comments. Also try to be consistent with style: `=` instead of `<-`, 4 spaces for indentation, and a space after a comma in function arguments, eg.

``` r
f(arg1, arg2)
```

instead of

``` r
f(arg1,arg2)
```

**Comment by vsluo** [comment]
hey changed the page according to your comments

**Comment by asnr** [comment]
Ace! Now merged.

**Pull request merged**